### PR TITLE
Adding markdown library and example in demo protocol.

### LINKED ISSUE #421: Update Information interface items to be markdown compatible
Information interface items that contain text should render markdown rather than being assumed to contain HTML.

There are a number of react markdown components that should be drop-in compatible, but we will need to ensure there isn't a significant performance impact since this will happen on-device.

### PR DESCRIPTION
**Pull request by jamieshark**
Resolves #421 and #354.

### CODE REVIEW

File: public/protocols/development.netcanvas/protocol.json [hunk 1]
```diff
@@ -658,11 +657,32 @@
       "type": "Information",
       "title": "Title for the screen",
       "items": [
-        { "type": "text", "content": "<p>Some arbitrary text content that can contain <strong>HTML</strong> tags.</p>" },
-        { "type": "audio", "content": "click_the_thing.mp3", "description": "Click by clicking the thing" },
+        {
+          "type": "text",
+          "content":
+            "Some arbitrary text content that can contain be **bold**"
```
Comment (rebeccamadsen) [inline public/protocols/development.netcanvas/protocol.json]: probably should be "can contain **bold**" or "can be **bold**"

**Comment by rebeccamadsen** [comment]
I get this error when running in android emulator:
```
Uncaught ReferenceError: require is not defined
    at Object.<anonymous> (main.e0e50ffd.js:31248)
    at __webpack_require__ (main.e0e50ffd.js:20)
    at Object.module.exports.Aacute (main.e0e50ffd.js:236863)
    at __webpack_require__ (main.e0e50ffd.js:20)
    at Object.<anonymous> (main.e0e50ffd.js:236842)
    at __webpack_require__ (main.e0e50ffd.js:20)
    at Object.exports.__esModule (main.e0e50ffd.js:235274)
    at __webpack_require__ (main.e0e50ffd.js:20)
    at Object.<anonymous> (main.e0e50ffd.js:235171)
    at __webpack_require__ (main.e0e50ffd.js:20)
    at Object.<anonymous> (main.e0e50ffd.js:232589)
    at __webpack_require__ (main.e0e50ffd.js:20)
    at Object.<anonymous> (main.e0e50ffd.js:65327)
    at __webpack_require__ (main.e0e50ffd.js:20)
    at Object.<anonymous> (main.e0e50ffd.js:21244)
    at __webpack_require__ (main.e0e50ffd.js:20)
```

**Comment by jthrilly** [comment]
I am currently implementing multiple webpack targets as a "proper" resolution to the issue we are having getting this to be runnable across Cordova and Electron. I will have this done very shortly.

**Comment by rebeccamadsen** [comment]
The education protocol is still using html in the information interfaces. That should be updated. Probably the wiki should be updated [here for sure](https://github.com/codaco/Network-Canvas/wiki/Information-Interface) and [possibly here](https://github.com/codaco/Network-Canvas/wiki/Name-Generator#interface-specific-api).

Otherwise this works for me!

**Pull request merged**